content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -12,6 +12,12 @@ weight: 10
 
 Client certificates generated by [kubeadm](/docs/reference/setup-tools/kubeadm/) expire after 1 year. This page explains how to manage certificate renewals with kubeadm.
 
+{{< warning >}}
+By default, kubeadm configures kubelet to renew its certificate automatically
+in `/var/lib/kubelet/pki/`, and it is not handled by `kubeadm certs` subcommands.
+The auto-renew process may sometime fail, see [below](#kubelet-client-cert).
```

Comment:
i don't like to include warnings such as "sometimes fail", unless we have a concrete explanation about the "why".

one scenario that is plausible in theory is:
- kubelet runs for 8 months
- the user shuts down the host
- restarts it after 4 months
- the certificate has expired

(but i don't think we should include such detail either.

see https://github.com/kubernetes/website/pull/28312#discussion_r646798474 for my recommendation instead of this "warning".

Reply:
Yeah I saw, work in progress, sorry I was interrupted, will commit shortly